sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/implementation/AesKeyCryptographyClient.java

```diff
@@ -28,28 +28,18 @@
 class AesKeyCryptographyClient extends LocalKeyCryptographyClient {
     private static final ClientLogger LOGGER = new ClientLogger(AesKeyCryptographyClient.class);
 
-    private byte[] key;
+    private byte[] aesKey;
 
     static final int AES_BLOCK_SIZE = 16;
 
-    AesKeyCryptographyClient(JsonWebKey key, CryptographyClientImpl implClient) {
-        super(implClient);
-        this.key = key.toAes().getEncoded();
-    }
+    AesKeyCryptographyClient(JsonWebKey jsonWebKey, CryptographyClientImpl implClient) {
+        super(jsonWebKey, implClient);
 
-    private byte[] getKey(JsonWebKey key) {
-        if (this.key == null) {
-            this.key = key.toAes().getEncoded();
-        }
-        return this.key;
+        this.aesKey = jsonWebKey.toAes().getEncoded();
```

Comment:
Should we throw an exception here when `aesKey` is null or empty instead of throwing it in the methods below?

Reply:
We cannot reach this point without making sure the JWK is `null` first. We make these checks in the public crypto client constructors (when a JWK is provided by the user) and when attempting to retrieve a JWK from the service (when a Key Vault `keyId` is provided).